htdocs/projet/tasks/task.php

```diff
@@ -619,9 +660,24 @@
 		$reshook = $hookmanager->executeHooks('addMoreActionsButtons', $parameters, $object, $action); // Note that $action and $object may have been
 		// modified by hook
 		if (empty($reshook)) {
+			// Cancel
+			if ($cancel) {
```

Comment:
Can you confirm we need this delete ?  If yes, can you describe the use case (steps in GUI when you reach this code) ?

Reply:
The way the code is implemented, this block is needed, as the user may cancel the cloning of the task.
Way to get to this block: 
- Click on a task
- Clone it
- Instead of saving the modifications, try to cancel. If we didn't had that block, the task would still have be cloned.